client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -62,7 +63,24 @@ class PacmanGame extends Component {
   }
 
   startGame = () => {
-    const { config, pacman } = this.state;
+    const { config, pacman, status } = this.state;
+    if (status === 1) {
+      let { score } = this.state;
+      clearInterval(this.animationHandler);
+      score -= 10;
+      this.setState({ status: 0, score });
+      return;
+    }
+    if (status === 2) {
+      this.setInitialGameState();
+      this.setState({
+        score: 0,
+        moveGhostsCount: 0,
+      });
+      this.setState({ status: 1 });
+    }
+    if (status === 0) this.setState({ status: 1 });
+    joinGame(pacman);
```

Comment:
Difficult to remember what status code is for what situation. Use a function or object instead.

Reply:
There's a comment in the State object of the PacmanGame component

Reply:
Yeah, that comment is non-authoritative. Replace that with a function or object.

Reply:
Yeah, that comment is non-authoritative. Replace that with a function or object.